src/CitizenProject/CitizenProjectManager.php

```diff
@@ -498,14 +498,51 @@ public function hasCitizenProjectInStatus(Adherent $adherent, array $status): bo
         return $this->getCitizenProjectRepository()->hasCitizenProjectInStatus($adherent, $status);
     }
 
+    /**
+     * Uploads and saves the citizen project image to the cloud storage.
```

Comment:
At this point, it does not matter if it's cloud or not, that's why we have an abstraction.